spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-repository-sample/src/main/java/com/example/Trader.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2017-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example;
+
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.google.cloud.spring.data.spanner.core.mapping.Column;
+import com.google.cloud.spring.data.spanner.core.mapping.Interleaved;
+import com.google.cloud.spring.data.spanner.core.mapping.PrimaryKey;
+import com.google.cloud.spring.data.spanner.core.mapping.Table;
+
+/**
+ * A sample entity.
+ *
+ * @author Mike Eltsufin
+ */
+@Table(name = "traders")
+public class Trader {
+	@PrimaryKey
+	@Column(name = "trader_id")
+	private String traderId;
+
+	@Column(name = "first_name")
+	private String firstName;
+
+	@Column(name = "last_name")
+	private String lastName;
+
+	@Column(name = "CREATED_ON")
+	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MMM-dd HH:mm:ss z")
+	private java.sql.Timestamp createdOn;
+
+	@Column(name = "MODIFIED_ON")
+	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MMM-dd HH:mm:ss z")
+	private List<java.sql.Timestamp> modifiedOn;
+
+	@Interleaved(lazy = true)
+	private List<com.example.Trade> trades;
+
+	public Trader() {
+	}
+
+	public Trader(String traderId, String firstName, String lastName) {
+		this.traderId = traderId;
+		this.firstName = firstName;
+		this.lastName = lastName;
+	}
+
+	public Trader(String traderId, String firstName, String lastName, Timestamp createdOn, List<Timestamp> modifiedOn) {
+		this.traderId = traderId;
+		this.firstName = firstName;
+		this.lastName = lastName;
+		this.createdOn = createdOn;
+		this.modifiedOn = modifiedOn;
+	}
+
+	public String getTraderId() {
+		return this.traderId;
+	}
+
+	public void setTraderId(String traderId) {
+		this.traderId = traderId;
+	}
+
+	public String getFirstName() {
+		return this.firstName;
+	}
+
+	public void setFirstName(String firstName) {
+		this.firstName = firstName;
+	}
+
+	public String getLastName() {
+		return this.lastName;
+	}
+
+	public void setLastName(String lastName) {
+		this.lastName = lastName;
+	}
+
+	public List<com.example.Trade> getTrades() {
+		return trades;
+	}
+
+	public void setTrades(List<com.example.Trade> trades) {
```

Comment:
Same thing in this class -- the `Trade` class is in the same package, so qualifying the class name with package name should not be necessary.